src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -139,7 +139,7 @@ public static IQueryable<Publication> HydratePublicationForPublicationViewModel(
                 }
 
                 // TODO: this will want updating when methodology status is fully implemented
```

Comment:
This TODO can be removed now

Reply:
Actually was deliberately keeping this TODO in place as a reminder that we still need to get the publisher to update the Methodology status and create a "Live" methodology status value as well as approved.